spec/fixtures/package.json

```diff
@@ -21,8 +21,7 @@
     "noflo-css": "^0.0.2"
   },
   "devDependencies": {
-    "noflo-runtime-iframe": "^0.8.0",
```

Comment:
Yes, it has the exact same `iframe` transport features, and these are being exercised in tests of this repo